### DISCUSSION TITLE
[Feature Request] Ability to set a fallback image for unsupported browsers when using WebP images

### DISCUSSION THREAD

**Discussion by jazir555** [elementor/elementor#25586] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Currently there is no way to manually set a fallback image for unsupported browsers in widgets that use images. Plugins like Shortpixel or others serve pngs or jpgs as fallbacks for WebP using the picture element. I compress my images manually as I can get my images compressed more than with automated converters. It would really help to be able to set a fallback image manually for users with older browsers.

### Describe the solution you'd like.

The ability to set a fallback image when a WebP image is chosen.

### Describe alternatives you've considered.

N/A

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by AWBroch** [comment]
This would be super helpful, to be able to properly support WebP without using another plugin that might not work well with Elementor, etc.

**Comment by m1ga** [comment]
https://caniuse.com/?search=WebP all browsers that Elementor supports (https://elementor.com/help/requirements/) can use WebP. If you use even older browsers or IE you have to add custom code/plugins as the note in the page says.
